unlock-protocol-com/blog/dashboard-redesign.md

```diff
@@ -0,0 +1,120 @@
+
+On this page, the lock manager can update settings for the lock, as well as see a list of all the members.
+
+### Customize NFT image
+
+By clicking the NFT image, a popup will allow you to update the NFT image. This can be done by uploading a file or via a URL.
+
+![Screenshot 2022-10-03 at 11.55.56.png](/images/blog/dashboard-redesign/Screenshot_2022-10-03_at_11.55.56.png)
+
+### Edit duration, quantity, and price
+
+Duration, quantity, or price information for a particular lock can be changed by clicking on the pencil next to the respective field in the left sidebar. Clicking the pencil to edit `Key Duration`, for example, will bring up a popup to edit the details of that field. By default, the current field value will be shown, which can then be edited. This same process works for updating `Quantity` and `Price` as well.
+
+![update-price.png](/images/blog/dashboard-redesign/update-price.png)
+
+### Members list
+
+The lock management page displays a list of members for the current lock, as well as metadata such as the expiration time for a particular key.
+
+![Screenshot 2022-10-03 at 12.04.31.png](/images/blog/dashboard-redesign/Screenshot_2022-10-03_at_12.04.31.png)
+
+Clicking on a key will expand it, and shows basic information about the key, owner, and expiration status.
+
+![Screenshot 2022-10-03 at 12.06.41.png](/images/blog/dashboard-redesign/Screenshot_2022-10-03_at_12.06.41.png)
+
+ In addition, if the viewer is the lock manager, more details and metadata for the key are shown.
+
+![Screenshot 2022-10-03 at 12.05.24.png](/images/blog/dashboard-redesign/Screenshot_2022-10-03_at_12.05.24.png)
+
+### Withdrawing funds from a lock 
+
+When the lock has a withdrawable amount, the `Withdraw` button will be active and allow the lock manager to withdraw the collected amount for the keys sold.
+
+![withdraw-amount.png](/images/blog/dashboard-redesign/withdraw-amount.png)
+
+### Generate purchase URL
+
+By clicking the `Generate URL` button, the user can generate a URL that can be shared with fans and allow them to easily purchase this membership.
+
+![generate-url.png](/images/blog/dashboard-redesign/generate-url.png)
+
+## Mobile experience
+
+The mobile experience is improved significantly with the new Dashboard. It features a fully responsive layout and provides a better experience and improved usability when using the app through a mobile device.
+
+### Change network and header 
+
+![mobile-header.png](/images/blog/dashboard-redesign/mobile-header.png)
+
+### Lock listing page
+
+![Screenshot 2022-09-28 at 13.16.57.png](/images/blog/dashboard-redesign/Screenshot_2022-09-28_at_13.16.57.png)
+
+### Manage lock page
+
+![manage-lock-mobile.png](/images/blog/dashboard-redesign/manage-lock-mobile.png)
+
+## See the new Dashboard now
+
+Ready to check it out? **[Try out the new Dashboard](https://app.unlock-protocol.com/dashboard)**.
```

Comment:
```suggestion
Ready to check it out? **[Try out the new Dashboard](https://app.unlock-protocol.com/locks)**.
```

Reply:
@julien51 based on the discussion in https://github.com/unlock-protocol/unlock/pull/9802 we eventually need to change with `/locks` path or keep both url

Reply:
ok thx